packages/devtools-reps/src/object-inspector/tests/component/expand.js

```diff
@@ -308,4 +308,61 @@ describe("ObjectInspector - state", () => {
 
     expect(formatObjectInspector(wrapper)).toMatchSnapshot();
   });
+
+  it("calls recordTelemetryEvent when expanding a node", async () => {
+    const recordTelemetryEvent = jest.fn();
+    const wrapper = mount(
+      ObjectInspector(
+        generateDefaults({
+          injectWaitService: true,
+          loadedProperties: new Map([
+            ["root-1", gripPropertiesStubs.get("proto-properties-symbols")]
+          ]),
+          recordTelemetryEvent,
+        })
+      )
+    );
+
+    const store = wrapper.instance().getStore();
+
+    let nodes = wrapper.find(".node");
+    const root1 = nodes.at(0);
+    const root2 = nodes.at(1);
+
+    // Expanding a node calls recordTelemetryEvent.
+    root1.simulate("click");
+    expect(recordTelemetryEvent.mock.calls).toHaveLength(1);
+    expect(recordTelemetryEvent.mock.calls[0][0]).toEqual("object_expanded");
```

Comment:
`expect(recordTelemetryEvent.mock.calls).toMatchSnapshot()` is probably fine as you likely just want to prevent regressions.

Again, no big deal.

Reply:
yeah, I think I like this way better because it better conveys what we want to test exactly

Reply:
sounds good